### PR TITLE
Victor martinez

### PR DESCRIPTION
**Pull request by hello202320**
Please provide insight on the profile page there's either some show only these components in Js or css. Component for book recommendations will not load.

### CODE REVIEW

**Comment by rishithab48** [comment]
Is that happening in the new profile pages?

**Comment by rishithab48** [comment]
Can you try having a separate page for it while I work on the profile pages?

File: src/pages/Stats/Stats.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React, { useState, useEffect } from 'react';
+import { useAuthState } from 'react-firebase-hooks/auth';
+import { auth, db } from '../../firebase';
+import { useNavigate, useParams } from 'react-router-dom';
+import RecommandBooks from '../../components/Recommandations/RecommandBooks';
```
Comment (racecn) [inline src/pages/Stats/Stats.js]: Can we fix the spelling of recommendations?

File: src/App.js [hunk 2]
```diff
@@ -97,7 +97,7 @@ const App = () => {
           {/* <Route path="/addbook" element={<AddBook/>}/> */}
           <Route path="/SearchResultsBook" element={<SearchResultsBook />} />
           <Route path="/ReaderQandA/:aid" element={<ReaderQandA />} />
-
+          <Route path="/stats/:uid" element={<Stats/>}></Route>
```
Comment (racecn) [inline src/App.js]: Is there a way to get to the stats page without manually typing this in?

File: src/pages/Profile.js [hunk 3]
```diff
@@ -15,13 +15,7 @@ const Profile = () => {
...
     const [show, setShow] = useState(false);
     const noShow = () => setShow(false);
-const g = {
-  action: 8.5,
-  comedy: 7.2,
-  drama: 9.1,
-  thriller: 8.0,
-  romance: 7.5
-}
+
```
Comment (rishithab48) [inline src/pages/Profile.js:18]: Old profile page doesn't have any recommendations anymore. Am I correct?

File: src/App.js
Comment (rishithab48) [inline src/App.js:1]: Will you also add a button for stats on the Sidebar so that the page is accessible to users?

File: src/pages/Stats/Stats.js
Comment (rishithab48) [inline src/pages/Stats/Stats.js:1]: Can we make the recommendations work dynamically now? I know you mentioned about making a function for it.

**Comment by rishithab48** [comment]
I also noticed that I can't see the latest changes you showed me in our last meeting for reading challenges. Did you not make commit it to thus branch? 
Screenshot 2024-04-10 215120

**Comment by rishithab48** [comment]
I also got errors when I tried to look at it with my UID. Only your UID worked.
Screenshot 2024-04-10 215324

**Review by racecn — APPROVED** [review]

**Pull request merged**